lpc55_sign_bin/src/main.rs

```diff
@@ -179,6 +179,43 @@ enum Command {
         #[clap(long)]
         debug_settings_cfg: PathBuf,
     },
+    /// Generate a request to sign a debug credential.
+    /// 
+    /// This is most useful when the debug credential needs to be signed by
+    /// a root key that is kept in a vault such as offline-keystore.
+    DebugCredentialSigningRequest {
+        out: PathBuf,
+
+        /// 16 byte UUID in hex.  If provided, the debug certificate will only
+        /// be sign challenges for a device that specifies the same UUID in its
+        /// debug authentication challenge.
+        #[clap(long)]
+        uuid: Option<String>,
+
+        /// File containing the private key that will be used to sign debug
+        /// authentication challenges.
+        #[clap(long)]
+        debug_key: PathBuf,
+
+        /// Must match Vendor Usage fields in CMPA/CFPA.  Acts as a revocation
+        /// scheme for debug credentials.
+        #[clap(long, default_value_t = 0)]
+        vendor_usage: u32,
+
+        /// When non-zero, ROM defers to running application when processing a
+        /// debug authentication response signed by this credential.  This
+        /// beacon value is provided to the running application to allow it to
+        /// decide what steps to take to prepare for debugging based on the
+        /// credential used.
+        #[clap(long, default_value_t = 0)]
+        beacon: u16,
+
+        /// TOML file containing debug access rights that the debug credential
+        /// will request.  These may not exceed the permissions specified by
+        /// CMPA and CFPA.
+        #[clap(long)]
+        debug_settings_cfg: PathBuf,
```

Comment:
Nit, as someone who's been running a lot of commands in this crate recently: maybe just `cfg`? The command lines get kind of verbose.